ntoskrnl/mm/mm.h

```diff
@@ -0,0 +1,2 @@
+#pragma once
+static inline VOID UpdateTotalCommittedPages(VOID);
```

Comment:
Why creating this file, instead of using the existing `ntoskrnl/include/internal/mm.h` ?
Also, it's in that .h that the implementation of the inline UpdateTotalCommittedPages() function must be placed.

Reply:
ok